frontend/src/views/Login.vue

```diff
@@ -1,29 +1,26 @@
 <script setup lang="ts">
 import identityApi from "@/services/api/identity";
-import storeAuth from "@/stores/auth";
 import storeHeartbeat from "@/stores/heartbeat";
 import type { Events } from "@/types/emitter";
 import type { Emitter } from "mitt";
-import { inject, onBeforeMount, ref } from "vue";
+import { inject, ref } from "vue";
 import { useRouter } from "vue-router";
 
 // Props
 const heartbeatStore = storeHeartbeat();
-const auth = storeAuth();
 const emitter = inject<Emitter<Events>>("emitter");
 const router = useRouter();
 const username = ref("");
 const password = ref("");
 const visiblePassword = ref(false);
 const logging = ref(false);
 
-function login() {
+async function login() {
   logging.value = true;
-  identityApi
+  await identityApi
```

Comment:
this form of using await and then is odd, don't worry about fixing it in this PR, i want to refactor those anyways

Reply:
yeah sounds good, we need to revisit the whole users management system anyways!